NEWS

```diff
@@ -3,6 +3,7 @@
 	* SHOGUN Release version 4.2.0 (libshogun 17.2, data 0.11, parameter 1)
 	* Features:
 	    - Add new Shogun cookbook [Heiko Strathmann, Sergey Lisitsyn]
+	    - Add new improved KMeans++ initialization [Saurabh Mahindre]	
```

Comment:
new improved is a bit vague... can you be concrete like "fixed bug in KMeans++" or "Improved efficiency of KMeans++"
It is also not a new feature but an efficiency improvement, there is a separate category for that